student-work/2023/Sergey-Boroditsky/api-final-project/get.md

```diff
@@ -0,0 +1,65 @@
+# GET Documentation
+
+### Description:
+Utilize the GPMD APIs to interact with the program's functionalities, such as placing orders and requesting bills. The GET method is utilized for initiating a bill request, while the POST method is used for creating a meal request. For further details about this API, refer to the README file.
```

Comment:
the word README should be a link to the readme file 
for example
[README](readme.md)